utils/contributors.js

```diff
@@ -59,6 +58,32 @@ const CONTRIBUTERS = [
         `,
         github: "https://github.com/debarati-06",
     },
+    {
+        handle_name: 'Satyam9119',
+        name: "Satyam Aditya",
+        image: "/images/Satyam.png",
+        description: `
```

Comment:
No image file.
If you prefer not to attach any image then remove the path line.